src/main/java/uk/gov/hmcts/darts/audio/model/ChannelAudio.java

```diff
@@ -20,6 +20,10 @@ public ChannelAudio(Integer channel) {
         this.channel = channel;
     }
 
+    public ChannelAudio(List<AudioFileInfo> audioFiles) {
```

Comment:
I'm a bit confused about what we're doing recently with the audio model files.
There is also `AudioSession` that has no usages.

Please can you check that we need all the lombok annotations on this `ChannelAudio` class - I think it is preferable in a custom class to use the required args constructor and getter methods just? Rather than having the setter methods as well?

`AudioFileInfo` - why do we need both fileName and path if they look to be the same?
Not sure we really want `@Setter` - it would be better to just use the constructor

I know we use the fileName for the audio operation commands currently. I wasn't sure if we were intending maybe to use the media.media_file, but perhaps that got forgotten about.

Reply:
AudioSession/ChannelAudio were added by ChrisM. I don’t know what the intended use is for AudioSession so will leave as is. ChannelAudio is in use so I will tidy up as per suggestion.

Tidied up AudioFileInfo - removed old fileName in favour of Path - plus other recommendations